src/Evaluator.php

```diff
@@ -11,14 +11,14 @@
 class Evaluator
 {
     private StatsigStore $store;
-    private Parser $ua_parser;
-    private IP3Country $ip3c;
+    private ?Parser $ua_parser;
+    private ?IP3Country $ip3c;
 
     function __construct(StatsigStore $store)
     {
         $this->store = $store;
-        $this->ua_parser = Parser::create();
-        $this->ip3c = new IP3Country();
+        $this->ua_parser = null;
```

Comment:
Now that is a breaking change!

Reply:
Its not - its just lazyily instantiated now.  All evaluation tests pass